isaric/parsers/isaric-rapid.toml

```diff
@@ -897,6 +903,15 @@
     { field = "oxygen_cmtype", values = { 2 = true }, description = "Oxygen therapy? (interface)" },
   ]
 
+[visit.treatment_oxygen_mask_prongs]
+  combinedType = "any"
+  fields = [
+    { field = "overall_oxygen_cmtype___1", values = { 1 = true, 0 = false }, description = "Nasal prongs" },
+    { field = "overall_oxygen_cmtype___3", values = { 1 = true, 0 = false }, description = "Mask" },
+    { field = "overall_oxygen_cmtype___4", values = { 1 = true, 0 = false }, description = "Mask with reservoir" },
+    { field = "oxygen_cmtype", values = { 1 = true, 3 = true, 4 = true }, description = "Oxygen therapy? (interface)" },
```

Comment:
@pipliggins the overall_oxygen_cmtype___ variables are checkboxes so I think best to only map the positive responses (0 responses are default entries where the checkbox was not ticked)